vnext/local-cli/generator-windows/templates/cs/src/App.xaml.cs

```diff
@@ -3,32 +3,33 @@
 namespace <%=ns%>
 {
     sealed partial class App : ReactApplication
+{
```

Comment:
looks like indentation got broken/wrong here?

Reply:
yeah, vscode keeps getting the wrong settings for our project and format it wrong. Haven't had time to figure out what setting is causing this...

---
In reply to: [426986471](https://github.com/microsoft/react-native-windows/pull/4947#discussion_r426986471) [](ancestors = 426986471)

Reply:
it was the <% in the namespace throwing vscode off....

---
In reply to: [427348372](https://github.com/microsoft/react-native-windows/pull/4947#discussion_r427348372) [](ancestors = 427348372,426986471)